science/volk-gnss-sdr/Portfile

```diff
@@ -94,11 +94,59 @@ depends_build-append port:pkgconfig
 depends_lib-append \
     path:lib/libvolk.dylib:volk \
     port:orc \
-    port:python27 \
-    port:py27-mako \
-    port:py27-six \
     port:boost
 
+# specify the Python dependencies; these are checked for at configure,
+# then used for building, but not at runtime.
+
+set pythons_suffixes {27 34 35 36 37 38}
```

Comment:
Let's not bother with Py34 or Py35 ... we're slowly dropping support for them, and keeping them only when these versions are the only ones available.

Reply:
Done!

Reply:
:+1: